todolist copy 3/main.js

```diff
@@ -1,111 +1,134 @@
-"use strict";
+'use strict'
 
-const addTaskButton = document.getElementById('addTask');
-const lineTaskInput = document.getElementById('taskInput');
-const toDoList = document.getElementById('tasklist');
-const clearing = document.getElementById('clearAll');
-const checkBoxCover = document.getElementById('clearAll');
+const addTaskButton = document.getElementById('addTask')
+const lineTaskInput = document.getElementById('taskInput')
+const toDoList = document.getElementById('task-list')
+const clearing = document.getElementById('clearAll')

```

Comment:
"clearing" is also a button
if we have addTaskButton, then this one needs to be clearAllButton